src/hooks/resources/device.ts

```diff
@@ -576,3 +589,59 @@ async function checkSupervisorReleaseUpgrades(
 		}
 	}
 }
+
+async function setSupervisorReleaseResource(
+	api: sbvrUtils.PinejsClient,
+	deviceIds: number[],
+	supervisorVersion: string,
+) {
+	// This function _should_ get called with one device (since each supervisor reports itself)
+	const device = (await api.get({
+		resource: 'device',
+		id: deviceIds[0],
+		options: {
+			$expand: ['is_of__device_type'],
+			$select: ['id', 'is_of__device_type', 'supervisor_version'],
+		},
+	})) as AnyObject;
+
+	if (device.supervisor_version != null) {
```

Comment:
Can you put this into a `$filter` above please

Reply:
done